gdrivefs/gdtool/drive.py

```diff
@@ -770,10 +770,15 @@ def remove_entry(self, normalized_entry):
 
         client = self.__auth.get_client()
 
-        args = { 'fileId': normalized_entry.id }
+        args = {
```

Comment:
We should modify the existing function (`remove_entry`). We should be adding a new function called `trash_entry`. These hit two different APIs. Just update the existing references from `remove_entry` to `trash_entry`. Thanks. We're almost there.